draft-ietf-httpbis-message-signatures.md

```diff
@@ -143,19 +143,19 @@ For brevity, the term "signature" on its own is used in this document to refer t
 In addition to those listed above, this document uses the following terms:
 
 Decimal String
-: 
+:
```

Comment:
```suggestion
: 
```

Reply:
here and elsewhere after ":", imho the extra space improves format: is that correct?

Reply:
What do you think those spaces do?

Reply:
With space, the HTML/TXT renders like

```
Signer
    The entity that is generating or has generated an HTTP Message Signature.
```

without space, it renders like

```
Signer
:    The entity that is generating or has generated an HTTP Message Signature.
```

Reply:
The correct way to do this is:

~~~
{: vspace="0"}
word:
: definition
~~~